test/REQUIRE

```diff
@@ -7,3 +7,4 @@ Ipopt
 ECOS
 SCS
 OffsetArrays 0.2.13
+PackageCompiler
```

Comment:
Since this isn't getting run in the automated tests, we probably don't need this in REQUIRE?